### DISCUSSION TITLE
Add Dark mode support in the sample app

### DISCUSSION THREAD

**Discussion by HobDev** [CommunityToolkit/Maui.Markup#78] [General]
There are docs for dark mode support with XAML in maui. But for someone using C# and C# markup to write the UI the sample should have support for dark mode.

**Comment by TheCodeTraveler** [comment]
Converting to Discussion

**Reply by bijington** [reply]
We already have #75. I think we could use that to cover dark mode support?

**Comment by HobDev** [comment]
But this is about implementing dark mode in the sample app until the AppThemeBinding is complete. I want to contribute if someone guide me how to do this.

**Reply by TheCodeTraveler** [reply]
You're welcome to open a PR for the sample app!

It's not a new feature to the toolkit so it doesn't require a New Feature Proposal. 

@bijington - what do you think about us creating another Issue Template for Sample App features/changes? We don't really have a workflow for the Sample App yet.

**Reply by HobDev** [reply]
@brminnick 
I have never done this before. I can implement the dark mode support by learning from the [GitTrend](https://github.com/brminnick/GitTrends). But this [bug](https://github.com/dotnet/maui/issues/8236) could be a blocker. As far as I can understand you are using the `Application.Current.RequestedThemeChanged event` to change the theme. I need some guidance about how you are changing the theme. Once I get hold of it I will implement it in the whole app.

**Reply by bijington** [reply]
@HobDev oh I see. I was thinking we could add support when making the change but I appreciate the two probably should be treated separately.

@brminnick I'm not sure we need a new template for changes to the sample app. My hope is this wouldn't happen often and typically changes will come with proposal implementations

**Reply by HobDev** [reply]
@bijington @brminnick I completed the darkmode support in the sample app. The link to the pull request is [this](https://github.com/CommunityToolkit/Maui.Markup/pull/98)
